easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.0-foss-2016b.eb

```diff
@@ -0,0 +1,62 @@
+name = 'QuantumESPRESSO'
+version = '6.0'
+
+homepage = 'http://www.pwscf.org/'
+description = """Quantum ESPRESSO  is an integrated suite of computer codes
+ for electronic-structure calculations and materials modeling at the nanoscale.
+ It is based on density-functional theory, plane waves, and pseudopotentials
+  (both norm-conserving and ultrasoft)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True, 'openmp': True}
```

Comment:
Yes, the other PR will work without it because LIBFFT and LIBFFT_MT are the same for MKL, for foss you end up with a link error because of a missing threaded lib. I'll add it here